src/main/java/com/google/sps/app/servlets/LogIn.java

```diff
@@ -27,10 +27,6 @@ public class LogIn extends HttpServlet {
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
         String username = request.getParameter(USER_NAME_PARAM).trim();
```

Comment:
Can you comment out the Validator instead of deleting the code, so that Mauricio can look at what the issue is.